minio/datatypes.py

```diff
@@ -755,3 +760,49 @@ def parse_copy_object(response):
     if last_modified:
         last_modified = from_iso8601utc(last_modified)
     return etag, last_modified
+
+
+class EventIterable:
+    """Context manager friendly event iterable."""
+
+    def __init__(self, func):
+        self._func = func
+        self._response = None
+        self._stream = None
+
+    def _close_response(self):
+        """Close response."""
+        if self._response:
+            self._response.close()
+            self._response.release_conn()
+            self._response = None
+
+    def __iter__(self):
+        return self
+
+    def _get_records(self):
+        """Get event records from response stream."""
+        try:
+            line = self._stream.next().strip()
```

Comment:
`self._stream` is a generator, which does not have a `next()` method. Instead using
```
line = next(self._stream).strip()
```
works.

Reply:
Thanks for testing the PR. I just modified to use `__next__()`. Hope this works fine.

Reply:
Yep, it still works.